src/app.js

```diff
@@ -1,27 +1,27 @@
-const path = require('path');
-const express = require('@feathersjs/express');
-const favicon = require('serve-favicon');
-const compress = require('compression');
-const cors = require('cors');
-const feathers = require('@feathersjs/feathers');
-const configuration = require('@feathersjs/configuration');
-const rest = require('@feathersjs/express/rest');
-const bodyParser = require('body-parser');
-const socketio = require('@feathersjs/socketio');
-const Sentry = require('@sentry/node');
+const path = require("path");
+const express = require("@feathersjs/express");
+const favicon = require("serve-favicon");
+const compress = require("compression");
+const cors = require("cors");
+const feathers = require("@feathersjs/feathers");
+const configuration = require("@feathersjs/configuration");
+const rest = require("@feathersjs/express/rest");
+const bodyParser = require("body-parser");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'body-parser'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)